spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -350,8 +350,11 @@ The default Task/Batch event and destination names on the broker are enumerated
 [[spring-cloud-dataflow-launch-tasks-from-stream]]
 == Launching Tasks from a Stream
 
-You can launch a task from a stream by using one of the available `task-launcher` sinks. Currently the only available
-`task-launcher` sink is the `task-launcher-local`, which will launch a task on your local machine.
+You can launch a task from a stream by using one of the available `task-launcher` sinks. Currently the platforms supported
+via the `task-launcher` sinks are
+https://github.com/spring-cloud/spring-cloud-stream-app-starters/tree/master/tasklauncher/spring-cloud-starter-stream-sink-tasklauncher-local[local],
```

Comment:
I think it should be https://github.com/spring-cloud-stream-app-starters/tasklauncher-local/tree/master/spring-cloud-starter-stream-sink-task-launcher-local